internal/endtoend/testdata/case_value_param/mysql/go/query.sql.go

```diff
@@ -0,0 +1,21 @@
+// Code generated by sqlc. DO NOT EDIT.
+// versions:
+//   sqlc v1.22.0
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+	"database/sql"
+)
+
+const update = `-- name: Update :exec
+UPDATE testing
+SET value = CASE ? WHEN true THEN 'Hello' WHEN false THEN 'Goodbye' ELSE value END
+`
+
+func (q *Queries) Update(ctx context.Context, value sql.NullString) error {
```

Comment:
Shouldn't this generated func parameter be a boolean to match the cases and not a string which matches the output?